roles/test_snmpTraps/tasks/snmpTraps_test.yml

```diff
@@ -0,0 +1,23 @@
+---
+# Assuming we've already logged in...
+# Following procedure on https://infrawatch.github.io/documentation/#creating-an-alert-rule-in-prometheus_assembly-advanced-features
+# Assuming we're in teh right project already...
```

Comment:
There needs to be some login tasks somewhere.

Failing that, if it is expected that the login is done in some separate script prior to the ``stf_functional_test.yml`` being run, then we need to add a test somewhere to make sure we're logged into the cluster. (I can do that somehow, and also document it).

Perhaps the most prudent is to add some pre-checks, and run them prior to the tests.
This could mean including a pre-check in the roles themselves, which would check that the correct vars are set, for example.
Additional pre-checks could take place in the ``stf_functional_tests.yml`` playbook, which would do some global tasks, like making sure the cluster exists, and that we're logged in.
The pre-checks are outside the scope of this PR, and should be included in a separate change. We can discuss the scope of the pre-checks.

Line 3 is not applicable here, and should either be removed or modified to include a reference to the snmp trap tests